components/ui/sticky-scroll-reveal.tsx

```
@@ -0,0 +1,103 @@
+  content,
+  contentClassName,
+}: {
+  content: {
+    title: string
+    description: string
+    content?: React.ReactNode
+  }[]
+  contentClassName?: string
+}) => {
+  const [activeCard, setActiveCard] = useState(0)
+  const { } = useScroll()
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)